reference/5.1/Microsoft.PowerShell.Management/Get-Clipboard.md

```diff
@@ -21,7 +21,8 @@ Get-Clipboard [-Format <ClipboardFormat>] [-TextFormatType <TextDataFormat>] [-R
 
 ## DESCRIPTION
 
-The `Get-Clipboard` cmdlet gets the current Windows clipboard entry.
+The `Get-Clipboard` cmdlet gets the current Windows clipboard entry. Multiple lines of text are
+returned as an array of strings similar `Get-Content`.
```

Comment:
Thanks for tackling this so quickly. 

Typo: "similar `Get-Content`" -> "similar to `Get-Content`"

Reply:
Doh! Fixed it.